### PR TITLE
FreeBSD mempools Fix

### PR DESCRIPTION
**Pull request by vitalisator**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

#### Fixes
On some (opnsense, pfsense, ...) FreeBSD based devices that using bsnmpd daemon, mempools show wrong value for hrStorageSize.
This fix use the real metrics from hrMemorySize.0 for affected devices.
This behaviour is only on bsnmpd based devices, net-snmp based are not affected.

### CODE REVIEW

File: includes/polling/mempools/hrstorage.inc.php [hunk 1]
```diff
@@ -12,5 +14,9 @@
 
 $mempool['units'] = $entry['hrStorageAllocationUnits'];
 $mempool['used']  = ($entry['hrStorageUsed'] * $mempool['units']);
-$mempool['total'] = ($entry['hrStorageSize'] * $mempool['units']);
+if ($device['sysObjectID'] == '.1.3.6.1.4.1.12325.1.1.2.1.1') { // bsnmpd based devices, like FreeBSD, opnsense, pfsense ...
```
Comment (murrant) [inline includes/polling/mempools/hrstorage.inc.php]: Maybe just check if hrMemorySize is good or hrStorageSize is bad instead of if this is *BSD?

Comment (vitalisator) [reply includes/polling/mempools/hrstorage.inc.php]: I have no idea how to evaluate if hrMemorySize is good or hrStorageSize is bad, both values are present and can be polled.
But I know that on *BSD the hrStorageSize values is wrong.

Comment (murrant) [reply includes/polling/mempools/hrstorage.inc.php]: So there are values there, they are just wrong?? That seems really odd.  What are the two values?

Comment (vitalisator) [reply includes/polling/mempools/hrstorage.inc.php]: here can you read some explanation to this behavior:
https://lists.freebsd.org/pipermail/freebsd-stable/2014-March/078146.html
So that is a old one, but still the same bug on bsnmpd

Comment (vitalisator) [reply includes/polling/mempools/hrstorage.inc.php]: in #7541 this issue was addressed too, but no one had the right solution :-)

**Comment by murrant** [comment]
@vitalisator fun, ok.  Would you be willing to add some test data for this since it is an edge case?
`./scripts/collect-snmp-data.php -h HOSTNAME`
